sql/src/test/java/io/crate/lucene/CIDRRangeQueryTest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.lucene;
+
+import io.crate.test.integration.CrateDummyClusterServiceUnitTest;
+import io.crate.testing.QueryTester;
+import org.elasticsearch.Version;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+public class CIDRRangeQueryTest extends CrateDummyClusterServiceUnitTest {
+
+    private static final String TABLE_NAME = "t";
+    private static final String COLUMN_NAME = "ip_addr";
+    private static final String CREATE_TABLE_STMT = String.format("create table %s (%s ip)", TABLE_NAME, COLUMN_NAME);
+
+    @Test
+    public void test_ipv4_cidr_operator() throws Throwable {
+        test(COLUMN_NAME,
+             new String[]{"192.168.0.1", "192.168.1.1", "192.168.1.7", "192.168.1.255", "192.168.2.0", "10.0.0.1"},
+             String.format("%s << '192.168.1.1/24'", COLUMN_NAME),
```

Comment:
Please don't use `String.format` without explicit `LOCALE`. We randomize the locales in tests, so this might be flaky.

I guess the column name could be inlined anyway, since it is static/always the same.